edb/server/compiler/explain.py

```diff
@@ -416,8 +422,15 @@ def analyze_explain_output(
 
     collapse_plan(plan)
 
+    config_vals = {
+        k: v for k, v in config_vals.items() if k not in OMITTED_CONFIG_VALS
```

Comment:
Maybe having a list of config names we want to include is a better strategy?

Reply:
I figured that a new option we will probably be more likely to want to include it than to not include it, and that we are likely to forget either way.